content.js

```diff
@@ -932,13 +932,13 @@ let tones = {
     5: ''
 };
 
-let utones = {
-    1: '\u0304',
-    2: '\u0301',
-    3: '\u030C',
-    4: '\u0300',
-    5: ''
-};
+// let utones = {
+//     1: '\u0304',
+//     2: '\u0301',
+//     3: '\u030C',
+//     4: '\u0300',
+//     5: ''
+// };
```

Comment:
Thanks for your changes. 👍 

I'm not sure about moving these though.
They seem to be used in non-zhuyin contexts too so they don't necessarily fit into that file.
Maybe @cschiller has a decision whether to include them.
(Either way, commented code should be removed.)

Reply:
Oops. I left the comments in! Doh!

Reply:
> Thanks for your changes. +1
> 
> I'm not sure about moving these though.
> They seem to be used in non-zhuyin contexts too so they don't necessarily fit into that file.
> Maybe @cschiller has a decision whether to include them.
> (Either way, commented code should be removed.)

I was wondering about this too. I was trying to avoid duplicate variables contain the same constants.